src/main/kotlin/no/nav/familie/ef/sak/behandling/migrering/MigreringService.kt

```diff
@@ -216,7 +218,9 @@ class MigreringService(
                     barn = behandlingBarn.map { it.id },
                     utgifter = periode.utgifterBarnetilsyn,
                     erMidlertidigOpphør = false,
-                    sanksjonsårsak = null
+                    sanksjonsårsak = null,
+                    periodetype = PeriodetypeBarnetilsyn.ORDINÆR,
+                    aktivitetstype = AktivitetstypeBarnetilsyn.I_ARBEID
```

Comment:
Burde aktivitetstype være null for migrerte saker? 
Samtidig kanskje det ikke har veldig mye å si? Burde ikke være veldig mange migreringer av Barnetilsyn 👀